Source/dolphin-memory-engine.vcxproj

```diff
@@ -126,7 +126,6 @@
     <ClCompile Include="GeneratedFiles\$(Configuration)\moc_MemWatchModel.cpp" />
     <ClCompile Include="GeneratedFiles\$(Configuration)\moc_MemWatchWidget.cpp" />
     <ClCompile Include="GeneratedFiles\$(Configuration)\moc_ResultsListModel.cpp" />
-    <ClCompile Include="GeneratedFiles\$(Configuration)\moc_AddressInputWidget.cpp" />
```

Comment:
Somehow this is part of your commit.
It's already merged as part of https://github.com/aldelaro5/dolphin-memory-engine/pull/165/files

Other than that looks good now, thanks.

Reply:
I'm sorry, I don't know how to remove this from the commit.

Reply:
Does `git rebase master` not work?
I saw the merge commit which resolved it, but then had the merge commit.

Reply:
If not using the dual remote strategy, go to your GitHub page:
https://github.com/jahorta/Dolphin-memory-engine/tree/master

and click "sync and DROP your own commits" (so it will revert to the current master). It should be a fairly obvious button on your page. Sync and Update? Sync and Discard? Sorry I don't recall the exact button words.

Then locally, delete your master branch:

```
git branch -D master
git fetch or git pull
git checkout master
git checkout fix_cmake_for_windows
git rebase master
git push -f
```

There are probably better/faster ways to do it, but this should resolve the issue.

Reply:
I tried using git rebase master when rebasing this branch and it still gave that line.

Reply:
Okay, I think I was able to get that to work. Thank you!

Reply:
Looks good to me. I'll do a test locally validating the paths are good and approve if everything checks out. Thanks